include/zephyr/drivers/led.h

```diff
@@ -128,6 +128,11 @@ __subsystem struct led_driver_api {
 	led_api_write_channels write_channels;
 };
 
+#ifdef CONFIG_LED_BLINK_FALLBACK
+int led_blink_fallback(const struct device *dev, uint32_t led, uint32_t delay_on,
+		       uint32_t delay_off);
```

Comment:
Suggestion:
```
#ifdef CONFIG_LED_BLINK_SOFTWARE
int led_blink_start(const struct device *dev, uint32_t led,
                       uint32_t delay_on, uint32_t delay_off);
int led_blink_stop(const struct device *dev, uint32_t led);
#else
#define led_blink_start(dev, led) do {} while (0)
#define led_blink_stop(dev, led) do {} while (0)
#endif
```

Or something like that...

Reply:
Unless we want to make them "public" API, I prefer to keep them out of led.h.
Since I move some functions to led_core.c, this is not needed here anymore.